packages/botfuel-dialog/src/brains/mongo-brain.js

```diff
@@ -119,17 +119,31 @@ class MongoBrain extends Brain {
     return user;
   }
 
-  /** @inheritdoc */
+  /**
+   * Sets a value for a key within the scope of the user
+   * @abstract
+   * @returns a JSON which contains the user id and the key that has just been set
```

Comment:
Here, the documentation reflects the fact that we don't return the same thing than the parent class documentation suggests.
Instead of returning the entire modified user object, which can be very heavy, we return only the id + modified key.
Does this looks OK for you ?

Reply:
Look goods to me, but we should be consistent with the MemoryBrain because all the brains should behave in the same way.